test/testdata/googleapis_test/gapic/lang/common.yaml

```diff
@@ -1,5 +1,6 @@
 common:
   toolkit_path: test/fake-repos/toolkit
+  staging_repo_dir: '${REPOROOT}/test/testdata/test_output/final'
```

Comment:
I don't see this being used in gapic_batch_staging_pipeline.baseline, should it be?